VERSION

```diff
@@ -1 +1 @@
-1.18.10
```

Comment:
Could we use git tags instead of the file?

Reply:
That would be brilliant, there is no real need for a version file

Reply:
We use both.

Reply:
The VERSION file is the SSOT for the version and it is used for CI/CD and tagging.